ext/pdo/pdo_dbh.stub.php

```diff
@@ -388,6 +388,13 @@ class PDO
 
     public function __construct(string $dsn, ?string $username = null, #[\SensitiveParameter] ?string $password = null, ?array $options = null) {}
 
+    public static function connect(
+        string $dsn,
+        ?string $username = null,
+        ?string $password = null,
+        ?array $options = null
+    ): PDO|PDOSqlite {}
```

Comment:
`|PDOSqlite` might be a leftover from initial attempts, I think they all extend PDO now (?)